form.php

```diff
@@ -1,321 +1,336 @@
-
-                if (FALSE === $result) {
-                    header('Location: error.php');
-                }
-            }
-
-            if (FALSE === $result) {
-                header('Location: error.php');
             }
 
-            $sql_query = "INSERT INTO `address`
-                (`employee_id`, `type`, `phone`, `fax`, `street`, `pin_no`, `city`, `state`)
-                VALUES ($employee_id, 'residence', '$r_phone', '$r_fax', '$r_street', '$r_pin',
-                '$r_city', '$r_state'),
-                ($employee_id, 'office', '$o_phone', '$o_fax', '$o_street', '$o_pin', '$o_city'
-                , '$o_state')";
-
-            $result = mysqli_query($conn, $sql_query);
-
-            if (FALSE === $result) {
-                header('Location: error.php');
-            }
-
-            header('Location: employee.php');
-        } else if (isset($_POST['update']) && isset($_GET['emp_id'])) {
-            $sql_query = "UPDATE `employee`
-                SET `first_name` = '$first_name', `middle_name` = '$middle_name',
-                `last_name` = '$last_name', `date_of_birth` = '$date_of_birth',
-                `prefix` = '$prefix', `photo` = '$photo', `note` = '$note',
-                `gender` = '$gender', `marital_status` = '$marital',
-                `communication` = '$communication', `employment` = '$employment',
-                `employer` = '$employer'
-                WHERE id = $emp_id";
-            $result = mysqli_query($conn, $sql_query);
-
-            if (FALSE === $result) {
-                header('Location: error.php');
-            }
-
-
-            $sql_query = "UPDATE `address`
-                SET `phone` = '$o_phone',`fax` = '$o_fax',
-                `street` = '$o_street',`pin_no` = '$o_pin',`city` = '$o_city',
-                `state` = '$o_state'
-                WHERE employee_id = $emp_id AND type = 'office'";
-            $result = mysqli_query($conn, $sql_query);
-
-            if (FALSE === $result) {
-                header('Location: error.php');
-            }
+        }
 
+    }
 
-            $sql_query = "UPDATE `address`
-                SET `phone`= '$r_phone',`fax` = '$r_fax',
-                `street`= '$r_street',`pin_no` = '$r_pin',`city` = '$r_city',
-                `state` = '$r_state'
-                WHERE employee_id = $emp_id AND type = 'residence'";
-            $result = mysqli_query($conn, $sql_query);
+   // If there is any error or not.
+   if ((0 === $error) &&  (FALSE === $valid->is_error()))
+   {
+        $times = '';
 
-            if (FALSE === $result) {
-                header('Location: error.php');
-            }
+        if (isset($_POST['submit']))
+        {
+            $times = 'first';
+        }
 
-            header('Location: employee.php');
+        if($employee->update_employee($_SESSION['emp_id'], $_POST,$times) &&
+            $db_obj->update('employee', "photo = '$photo'", 'where id = ' . $_SESSION['emp_id']))
```

Comment:
both the update queries can be made on new lines and can be assigned to variables